### DISCUSSION TITLE
When will the new docs' MD files open source?

### DISCUSSION THREAD

**Discussion by utakotoba** [vercel/next.js#44352] [Help]
Hi, I'd like to **translate the new docs into my spoken language** (Chinese), so I'd like to know when will **the official make the new(beta) docs' MDs open to public like the current version**.

A few month ago I asked Vercel's CSE(Customers Support Engineer) about whether the official will launch a new project to translate the docs, while he didn't reply confirmly, that's OK...

So i'd like to translate by my self, well, my project has a [previous repo](https://github.com/ifshizuku/nextjs-zh-docs), but it has been archived by myself before as a result of the **pressure of my school**.

But now I've time, so I come back to ask if the official or community have the plan to do translations(to any language).

Thanks!

**Comment by mononoke-choi** [comment]
i prefer to know all changes to catch up new version. so i'm looking forward to it too.
